WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -128,6 +128,12 @@ protected override void OnGotFocus(GotFocusEventArgs e)
 
 	protected override void OnTextInput(TextInputEventArgs e)
 	{
+		if (Text.Trim() == "" && e.Text == ".")
```

Comment:
Are we sure `Text` cannot be null? If not, `string.IsNullOrWhiteSpace()` instead of `== ""`?